tmt/plans/integration.fmf

```diff
@@ -0,0 +1,56 @@
+# Please change the image when you run this plan locally with tmt run
+# tmt run -vvvvv plan -n /integration/test-01-readonly
+# local image: file:///home/foobar/image.qcow2
+provision:
+  how: virtual
+  image: https://composes.stream.centos.org/stream-10/production/CentOS-Stream-10-20250331.0/compose/BaseOS/x86_64/images/CentOS-Stream-GenericCloud-10-20250331.0.x86_64.qcow2
```

Comment:
Just circling back to the details here...this is going to get garbage collected at some point, we need to change to use a floating image *or* bump via renovate (noisy)

Reply:
Yeah, this one is for local run only, testing farm has its own image to deploy on TF runner. I left this option here for easy running locally.